katello/katello-debug.sh

```diff
@@ -81,14 +84,9 @@ add_cmd "qpid-stat --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b
 add_files /etc/gofer
 add_files /var/log/gofer
 
-#foreman-tasks export
-if hash foreman-rake 2>/dev/null; then
-  echo "Exporting tasks, this may take a few minutes."
-  tasks_filename=`foreman-rake foreman_tasks:export_tasks 2> /tmp/tasks_export.log | tail -n 1 | awk '{print $2}'`
-  copy_files $tasks_filename
-  add_files /tmp/tasks_export.log
-  rm -f $tasks_filename /tmp/tasks_export.log
-fi
+# Foreman Tasks fast export (for HTML version use foreman-rake foreman_tasks:export_tasks)
+/usr/bin/pg_dump -- foreman -a -t dynflow_\* -t foreman_tasks_\* | bzip2 -9 > /tmp/tasks-db-dump.sql.bz2
```

Comment:
you need to have "su postgres -s" otherwise the command returns

    pg_dump: too many command-line arguments (first is "-a")